src/pipeline_edsl/to_workflow.ml

```diff
@@ -410,6 +424,36 @@ module Make (Config : Compiler_configuration)
       MHC_alleles node
 
 
+  let kallisto ~reference_build ?(bootstrap_samples=100) fastq =
+    let result_prefix =
+      Name_file.in_directory ~readable_suffix:"kallisto" Config.work_dir [
+        fastq#product#escaped_sample_name;
+        sprintf "%d" bootstrap_samples;
+        reference_build;
+      ] in
+    Kallisto_result (
+      Tools.Kallisto.run
+        ~reference_build ~processors:1
```

Comment:
We should get the `processors` from the Machine.
https://github.com/hammerlab/biokepi/blob/master/src/run_environment/machine.ml#L199